doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -2992,9 +2996,36 @@ Deletes an ``AuthenticatedUser`` whose ``id``  is passed. ::
 
     DELETE http://$SERVER/api/admin/authenticatedUsers/id/$id
     
-Note: If the user has performed certain actions such as creating or contributing to a Dataset or downloading a file they cannot be deleted.
+Note: If the user has performed certain actions such as creating or contributing to a Dataset or downloading a file they cannot be deleted. If a user cannot be deleted for this reason, you can choose to :ref:`disable a user<disable-a-user>`.
```

Comment:
I don't know where this belongs but (assuming we still want it) we should should document the "get user traces" API and explain that it can reveal details on the "certain actions" above. My thought is that it could be a starting point to understand a user's impact and from there a decision can be made to delete or disable.

Reply:
Yes, we still want the user traces API. Let me know if you'd like me to document it using some copy/paste from another API or if you'd like to do so.